packages/a11y-testing/src/definitions/react-button/buttonAccessibilityBehaviorDefinition.ts

```diff
@@ -0,0 +1,76 @@
+import { Rule } from './../../types';
+import { BehaviorRule } from './../../rules/rules';
+
+export const buttonAccessibilityBehaviorDefinition: Rule[] = [
+  BehaviorRule.root()
+    .doesNotHaveAttribute('role')
+    .doesNotHaveAttribute('tabindex')
+    .description(`if element is rendered as a default 'button'.`),
+  // BehaviorRule.root()
+  //   .forProps({ href: '#' })
+  //   .hasAttribute('role', 'button')
+  //   .hasAttribute('tabindex', '0')
+  //   .description(`if element has href and is rendered as an 'anchor'.`),
```

Comment:
Just wanted to know that I've commented for now the tests that reference commented props.

Reply:
might be worth adding that somewhere in this definition